src/AppBundle/Service/JobHandler.php

```diff
@@ -131,9 +133,15 @@ public function handleStaleJobs(Event $event): void
             return;
         }
 
+        $dayAgo = new DateTime('-1 day');
         /** @var Job $job */
         foreach ($staleJobs->getIterator() as $job) {
-            $event->removeJob($job);
+            if ($job->isBusy()) {
+                $job->setStatus(Job::STATUS_FAILED_TIMEOUT);
+            }
+            if ($job->getSubmitted() >= $dayAgo) {
+                $event->removeJob($job);
```

Comment:
`Job::isBusy()` gets Jobs that are either queued or started. The only other states are failed states, those are left intentionally so the user can see them. So I guess we don't need to remove the Job at all.

Reply:
This just makes sure that very old jobs don't get stored forever.

Reply:
Stored forever is fine, I think... If I leave a job running, and come back the next day, I might wonder why nothing happened even though I recall having initiated an update. I don't have any strong feelings, just a thought.